src/opl.c

```diff
@@ -191,6 +191,7 @@ unsigned char gDefaultUITextColor[3];
 hdl_game_info_t *gAutoLaunchGame;
 char gOPLPart[128];
 char *gHDDPrefix;
+char gExportName[32];
```

Comment:
This new parameter does not have a default value, and is also never initialized. Shouldn't this be at least initialized to 0?